Add a table with standard temperatures temperatures raw and Celsius values (#391)
icub-tech-iit/documentation: docs/temperature_sensors/software/reading_data.md

```

> ### Typical temperature values table

Reviewer: ```suggestion
### Table of typical temperature values
```

> In order to help the user to understand the raw values of the temperatures when read in the `yarprobotinterface` logs we have decided to provide this table with a good range of temperatures. 

Reviewer: ```suggestion
Look up this table containing a good range of temperatures to better interpret the raw temperature values reported within the `yarprobotinterface` logs. 
```

> Since now in the robot the motors can mount either temperature sensor PT100 or PT1000 we show the correlation between raw and Celsius values for both of them:

Reviewer: ```suggestion
In particular, our robots can mount either PT100 or PT1000 temperature sensors, thus find below the correlation between raw and Celsius values for both sensors:
```
